cmd/bucket-handlers.go

```diff
@@ -17,6 +17,7 @@
 package cmd
 
 import (
+	"github.com/minio/minio-go/pkg/set"
```

Comment:
Don't use this style of importing follow this https://github.com/golang/go/wiki/CodeReviewComments#imports